components/flipper.py

```diff
@@ -2,18 +2,28 @@
 
 
 class Flipper:
+    #this flips a bucket with an arm that has a spinning motor
+    
 
     flipper_motor = wpilib.Talon
+    # a talon controls a motor
     spinning = False
+    #It starts not spinning
 
     def turn_on(self):
+        '''defines "on"  '''
         self.spinning = True
+    
 
     def turn_off(self):
+    '''defines "off"'''
```

Comment:
You will need to indent this docstring, Python requires an indented block directly after a function declaration.

Reply:
Double quotes for this docstring too.